src/d3a/d3a_core/export.py

```diff
@@ -309,26 +352,28 @@ def plot_device_stats(self, area: Area, node_address_list: list):
                 address_list = new_node_address_list + [child.name]
                 self._plot_device_stats(address_list, child.strategy)
 
+    @staticmethod
+    def _get_from_dict(data_dict: Dict, map_list: List) -> Mapping:
+        """Get data from dict by following a map_list."""
```

Comment:
Nit: I think the docstring might be further improved by providing an example.
```suggestion
        """Get nested data from a dict by following the keys provided in a list.
        
        >>> dictionary = {"Parent 1": {"Child 1": "value 1"}, "Parent 2": {"Child 2": "value 2"}}
        >>> keys_path = ["Parent 2", "Child 2"]
        >>> _get_from_dict(dictionary, keys_path)
        >>> 'value 2'
        """
```
(This is the [doctest syntax](https://docs.python.org/3/library/doctest.html))